### PR TITLE
Added benchmarks scripts with small instruction

### LINKED ISSUE #8712: Document how to use SQL scripts to create benchmark databases

### PR DESCRIPTION
**Pull request by andriikorotkov**
## What
*Describe what the change is solving*
*It helps to add screenshots if it affects the frontend.*

## How
added benchmarks scripts with small instruction

## Recommended reading order
1. `x.java`
2. `y.python`

## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.

## Pre-merge Checklist
Expand the relevant checklist and delete the others. 

<details><summary> <strong> New Connector </strong></summary>
<p>

#### Community member or Airbyter
   
- [ ] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] `docs/SUMMARY.md`
    - [ ] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
   
</p>
</details>

<details><summary> <strong> Updating a connector </strong></summary>
<p>
   
#### Community member or Airbyter
   
- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the new connector version is published, connector version bumped in the seed directory as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</p>
</details>

<details><summary> <strong> Connector Generator </strong> </summary>
<p>
   
- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed.
</p>
</details>

### CODE REVIEW

File: docs/benchmarks-scripts/mssql/mssql-script.sql
Comment (tuliren) [inline docs/benchmarks-scripts/mssql/mssql-script.sql:247]:
```suggestion
-- TODO: change the value to control the number of tables
EXEC table_copy @tablecount = 1;
```

Instead of taking screenshots, I think it is more update-proof to `TODO`s as comments in the script itself to guide users. In this way, when the script is changed, the screenshots do not need to be updated accordingly.

File: docs/benchmarks-scripts/mssql/mssql-script.sql
Comment (tuliren) [inline docs/benchmarks-scripts/mssql/mssql-script.sql:239]:
```suggestion
-- TODO: change the following @allrows to control the number of records with different sizes
-- number of 50B records
EXEC insert_rows @allrows = 0, @insertcount = 998, @value = @extraSmallText
-- number of 500B records
EXEC insert_rows @allrows = 0, @insertcount = 998, @value = @smallText
-- number of 10KB records
EXEC insert_rows @allrows = 0, @insertcount = 998, @value = @regularText
-- number of 100KB records
EXEC insert_rows @allrows = 0, @insertcount = 98, @value = @largeText
```

File: docs/benchmarks-scripts/mssql/mssql-script.md
Comment (tuliren) [inline docs/benchmarks-scripts/mssql/mssql-script.md:14]: After moving the contents of the screenshots to `TODO` comments in the scripts, the above bullets can be updated to:

> 2. Follow the TODOs in **mssql-script.sql** to change the number of tables, and the number of records of different sizes.

**Review by tuliren — COMMENTED** [review]
Looks great!

I have two suggestions:

1. The documentation and the script can be moved under each source database module.

    Take `MySQL` as an example:
    - The documentation can be moved to `airbyte-integrations/connectors/source-mysql/README.md`.
    - The SQL script can be moved to `airbyte-integrations/connectors/source-mysql/src/test-performance/sql/*.sql`.

2. The contents of the screenshots should be replaced with `TODO`s as comments in the scripts. In this way, it is easier to find which line of code to modify (rather than searching for them based on the screenshots).

File: docs/benchmarks-scripts/mssql/mssql-script.md
Comment (tuliren) [inline docs/benchmarks-scripts/mssql/mssql-script.md:15]: Can you also include the command to run the script against the database?

File: docs/benchmarks-scripts/mssql/mssql-script.sql
Comment (andriikorotkov) [reply docs/benchmarks-scripts/mssql/mssql-script.sql]: I still cannot put scripts in the specified folders, they will be added only when Eugene's changes are merged

File: docs/benchmarks-scripts/mssql/mssql-script.md
Comment (andriikorotkov) [reply docs/benchmarks-scripts/mssql/mssql-script.md]: For each instruction, I added a line how to run the script from the command line

**Comment by andriikorotkov** [comment]
@tuliren, I added the changes you mentioned. I propose to merge this pull request after we merge Eugene's pull request.

File: airbyte-integrations/connectors/source-mssql/mssql-script.sql
Comment (tuliren) [inline airbyte-integrations/connectors/source-mssql/mssql-script.sql:119]: Can you use the [REPLICATE](https://docs.microsoft.com/en-us/sql/t-sql/functions/replicate-transact-sql?view=sql-server-ver15) function to construct these long string? Especially for `@regularText` and `@largeText`.

File: airbyte-integrations/connectors/source-mssql/README.md
Comment (tuliren) [inline airbyte-integrations/connectors/source-mssql/README.md:1]: Nitpick. I'd suggest to give this doc a generic H1 title, because this is a README for the entire module. You can move what you wrote under a performance test H2 title. Something like:

```suggestion
# MsSQL (SQL Server) Source

## Performance Test

### Use MsSQL script to populate the benchmark database
```

File: airbyte-integrations/connectors/source-mysql/mysql-script.sql
Comment (tuliren) [inline airbyte-integrations/connectors/source-mysql/mysql-script.sql:112]: For MySQL, you can use the [REPEAT](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_repeat) function to generate these repetitive strings.

File: airbyte-integrations/connectors/source-postgres/README.md
Comment (tuliren) [inline airbyte-integrations/connectors/source-postgres/README.md:1]: Similar as MsSQL, it's better to put this under a performance test H2 title.

```suggestion
# Postgres Source

## Performance Test

### Use Postgres script to populate the benchmark database
```

File: airbyte-integrations/connectors/source-postgres/3-run-script.sql
Comment (tuliren) [inline airbyte-integrations/connectors/source-postgres/3-run-script.sql:20]: For Postgres, there is also a [repeat](https://www.postgresql.org/docs/9.1/functions-string.html) function you can use.

Comment (andriikorotkov) [reply airbyte-integrations/connectors/source-postgres/3-run-script.sql]: @tuliren, thanks, those are great comments. I corrected the script and changed it so that the insert adds the exact number of records.

File: airbyte-integrations/connectors/source-postgres/README.md
Comment (andriikorotkov) [reply airbyte-integrations/connectors/source-postgres/README.md]: updated it

File: airbyte-integrations/connectors/source-mssql/mssql-script.sql
Comment (andriikorotkov) [reply airbyte-integrations/connectors/source-mssql/mssql-script.sql]: updated it

File: airbyte-integrations/connectors/source-mysql/mysql-script.sql
Comment (andriikorotkov) [reply airbyte-integrations/connectors/source-mysql/mysql-script.sql]: updated it

File: airbyte-integrations/connectors/source-mssql/README.md
Comment (andriikorotkov) [reply airbyte-integrations/connectors/source-mssql/README.md]: updated it

**Review by tuliren — APPROVED** [review]
🎉

**Pull request merged**
